jscomp/frontend/external_ffi_types.pp.ml

```diff
@@ -223,7 +224,7 @@ let check_ffi ?loc ffi : bool =
       -> 
       upgrade (is_package_relative_path external_module_name.bundle);
       check_external_module_name external_module_name
-    | Js_new {external_module_name ;  name; scopes = _}
+    | Js_new {external_module_name ;  name; splice = _; scopes = _}
```

Comment:
Not sure why unused fields are emphasised in this area of code.
Is that a reminder that they are there, but unused?

Reply:
I'm not sure too but my guess is: if we add an additional member (like `splice` in this PR), it will result in a warning 9 (missing-record-field-pattern), so we can easily notice that some modification may be needed here.